plugins/moodbar/__init__.py

```diff
@@ -543,20 +608,41 @@ def enable(exaile):
     else:
         _enable(None, exaile, None)
 
+
 def _enable(eventname, exaile, nothing):
-    global ExaileModbar
-    ExaileModbar.readMod('')
-    ExaileModbar.setupUi()
-    event.add_callback(ExaileModbar.play_start, 'playback_track_start', player.PLAYER)
-    event.add_callback(ExaileModbar.play_end, 'playback_player_end', player.PLAYER)
+    _enable_main_moodbar(exaile)
+
+    event.add_callback(_enable_preview_moodbar, 'preview_device_enabled')
+    event.add_callback(_disable_preview_moodbar, 'preview_device_disabling')
+
+    try:
+        import previewdevice
```

Comment:
I'd rather add a function to the plugin manager instead of accessing the dictionary directly, but this definitely seems better.